### DISCUSSION TITLE
Message: Cannot connect to SQL Server Browser. Ensure SQL Server Browser has been started.

### DISCUSSION THREAD

**Discussion by mateuslfreitas** [microsoft/azuredatastudio#17324] [Q&A]
HI! 
I'm trying to connect to a remote instance of sqlserver and AzureDataStudio returns with this message:
"Cannot connect to SQL Server Browser. Ensure SQL Server Browser has been started."

What am I doing wrong?

Thank's

I'm running on a Mac OS X Mojave

**Comment by kburtram** [comment]
@mateuslfreitas SQL Server Browser is a different service that is installed as part of SQL Server.  Typically the relational engine runs on port 1433 and the browser will run on 1434 (and be disabled by default).  Can you provide some more details how you are connecting to SQL Server?  for example, are you using IP address or DNS name?  Are you specifying a port?  Is SQL Server running on the default port on the server? etc.
